Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/resources/Templates/SidebarForLogin.xhtml

```diff
@@ -37,9 +37,11 @@
     <ui:composition>
         <div class="col-xs-2 sidebar">
             <h1 class="page-header">Gateway Administrative Console</h1>
-            <div align="center" class="sidebar-gfx">
-                <img src="resources/images/gfx-consoleiconic.png" class="img-responsive" />
-            </div>
+           <div  class="sidebar-gfx"  >
+             <p class="text-center">
+                <img src="resources/images/gfx-consoleiconic.png" class="img-responsive" alt="gfx-consoleiconic"/>
```

Comment:
The alt text needs to be something more descriptive.  Not just the name of the file.  Something like:  "Admin GUI Login Icon"